### PR TITLE
Added fortnite-api presence

### PR DESCRIPTION
**Pull request by Leafageee**
Made a presence for: https://fortnite-api.com/

First presence so its 💩

### CODE REVIEW

**Comment by Bas950** [comment]
https://github.com/PreMiD/Presences/blob/master/CONTRIBUTING.md

**Comment by Bas950** [comment]
Please sent screenshots as proof that it is working

File: Fortnite-API/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	"description": {
+		"en": "Stop paying for outdated API's. We provide a reliable API with always updated content."
+	},
+	"url": [
+		"https://fortnite-api.com",
+		"fortnite-api.com"
+	],
+	"version": "0.1",
+	"logo": "https://cdn.discordapp.com/attachments/634078451841761286/648591677748936704/logo.png",
+	"thumbnail": "https://prnt.sc/q1x3at",
```
Comment (Bas950) [inline Fortnite-API/dist/metadata.json]: Upload both logo and thumbnail to a site like imgur and then use the .png link

**Comment by Leafageee** [comment]
> Please sent screenshots as proof that it is working

image
image
image

File: Fortnite-API/dist/metadata.json (hunk 1 above)
Comment (Leafageee) [reply Fortnite-API/dist/metadata.json]: K, i'll change them now

Comment (Leafageee) [reply Fortnite-API/dist/metadata.json]: Changed

**Pull request merged**